File: 37 -  Sum of Digits Until Single Digit/solution.java
public class solution {
    public static int sumOfDigits(int num) {
        while ( num >= 10 ) {
            int sum = 0;
            while ( num > 0 ) {
                sum += num % 10;
                num = num / 10;
            }
            num = sum;
        }
        return num;
    }



    public static void main(String[] args) {
        System.out.println(sumOfDigits(38));
    }
}

File: 37 -  Sum of Digits Until Single Digit/solution.js
function sumOfDigits(num) {
  // Write your code here
  while (num >= 10) {
    let sum = 0;
    while (num > 0) {
      sum += num % 10;
      num = Math.floor(num / 10);
    }

    num = sum;
  }

  return num;
}

// Test Cases
console.log(sumOfDigits(38));
